src/main/drivers/bus_quadspi.c

```diff
@@ -158,6 +158,64 @@ const quadSpiHardware_t quadSpiHardware[] = {
         .rcc = RCC_AHB3(QSPI),
     },
 #endif
+#ifdef STM32G4
+    {
+        .device = QUADSPIDEV_1,
+        .reg = QUADSPI,
+        .clkPins = {
+            { DEFIO_TAG_E(PA3), GPIO_AF10_QUADSPI },
```

Comment:
Somo MCUs have one AF shared for all peripheral pins. IIRC g4 is one of them (see UART) - in that case  GPIO_AF10_QUADSPI should be property of quadSpiHardware, not individual pins (it will need a bit more refactoring/path splitting in qspi driver. But it can save repetition if new hardware is added